lib/grant-types/refresh-token-grant-type.js

```diff
@@ -101,7 +101,7 @@ RefreshTokenGrantType.prototype.getRefreshToken = function(request, client) {
       }
 
       if (token.client.id !== client.id) {
-        throw new InvalidGrantError('Invalid grant: refresh token is invalid');
+        throw new InvalidGrantError('Invalid grant: refresh token is invalid because token.client.id !== client.id');
```

Comment:
```suggestion
        throw new InvalidGrantError('Invalid grant: refresh token was issued to another client');
```

Reply:
This error message is closer to the spec in https://tools.ietf.org/html/rfc6749#section-5.2 where it says:

> The provided authorization grant (e.g., authorization code, resource owner credentials) or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or **was issued to another client**.

Reply:
perfect